.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: git lfs checkout
       - uses: pnpm/action-setup@v2
         with:
-          version: 7
+          version: 8
```

Comment:
in package.json at the root, it says v7
pick one or the other

Reply:
same goes for line 66

Reply:
It should be 8.